Tests/GoogleAITests/GenerativeModelTests.swift

```diff
@@ -400,7 +385,15 @@ final class GenerativeModelTests: XCTestCase {
   }
 
   func testGenerateContentMissingSafetyRatings() async throws {
-    try await internalTestGenerateContent(resource: "MissingSafetyRatings", safetyRatingsCount: 0)
+    MockURLProtocol.requestHandler = try httpRequestHandler(
+      forResource: "unary-success-missing-safety-ratings",
+      withExtension: "json"
+    )
+
+    let content = try await model.generateContent(testPrompt)
+    let promptFeedback = try XCTUnwrap(content.promptFeedback)
+    XCTAssertEqual(promptFeedback.safetyRatings.count, 0)
+    XCTAssertNotNil(content.text)
```

Comment:
Why not an `XCTAssertEqual` here?

Reply:
Updated, the text was quite long in the test file so I've  shortened it and added an `XCTAssertEqual`